pkgs/applications/virtualization/driver/win-virtio/default.nix

```diff
@@ -1,18 +1,20 @@
 { lib, stdenv, fetchurl, p7zip }:
 stdenv.mkDerivation rec {
   pname = "win-virtio";
-  version = "0.1.141-1";
+  version = "0.1.196-1";
 
   phases = [ "buildPhase" "installPhase" ];
```

Comment:
phases should not be overwriten.

Reply:
Okay, I've replaced packagePhase instead. Not sure if the prePackage/postPackage hooks are necessary though since it's a noop.

Reply:
> Not sure if the prePackage/postPackage hooks are necessary though since it's a noop.

They should be added for new packages and updates.